src/runner.rs

```diff
@@ -958,15 +942,12 @@ impl Log for RunnerLogger {
 
 static mut LOGGER: mem::MaybeUninit<RunnerLogger> = mem::MaybeUninit::uninit();
 
-pub fn get_runner_logger() -> &'static impl Log {
+pub fn get_runner_logger(log_file_path: Option<PathBuf>) -> &'static impl Log {
     static INIT: Once = Once::new();
 
     unsafe {
         INIT.call_once(|| {
-            let local_offset =
-                UtcOffset::current_local_offset().expect("failed to get local time offset");
-
-            LOGGER.write(RunnerLogger { local_offset });
+            LOGGER.write(RunnerLogger::new(log_file_path).expect("failed to set logger"));
```

Comment:
Let's not panic if the runner can't be initialized, and instead handle the Result.